### PR TITLE
extend CI check

### PR DESCRIPTION
**Pull request by niclim**
## 🍗 Description
_What does this PR do? Anything folks should know?_

I threw in more cases like `ci=1` or `ci=yes` or `ci=y` but im not sure what else we might want to cover here

## 📚 References
_Links to relevant docs (Notion, Twist, GH issues, etc.), if applicable._

## 👹 QA
_How can other humans verify that this PR is correct?_

### CODE REVIEW

File: projects/optic/src/config.ts [hunk 1]
```diff
@@ -32,7 +32,11 @@ const DefaultOpticCliConfig: OpticCliConfig = {
   ruleset: undefined,
   isAuthenticated: false,
   client: createOpticClient('no_token'),
-  isInCi: process.env.CI === 'true',
+  isInCi:
+    process.env.CI === 'true' ||
+    process.env.CI === '1' ||
+    process.env.CI === 'yes' ||
+    process.env.CI === 'y',
```
Comment (notnmeyer) [inline projects/optic/src/config.ts:39]: are these case sensitive?

**Comment by notnmeyer** [comment]
here are yaml's boolean values: https://yaml.org/type/bool.html

i think we should use 'em

**Comment by notnmeyer** [comment]
oh, and we should _never_ suggest in documentation that anyone use any value for the `CI` env var other than "true". since `CI` is set automatically by most (all?) CI providers deviating from the default is here-be-dragons territory.

**Comment by niclim** [comment]
Yeah totally, this is just to catch any case where it's set somewhere else inadvertently

**Pull request merged**